### PR TITLE
Typo + Some Suggestion For Wrapping and breaking text guide

### PR DESCRIPTION
**Pull request by digi-booster**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

https://github.com/mdn/content/issues/7904
> What was wrong/why is this fix needed? (quick summary only)

hyphen typo

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ In CSS, if you have an unbreakable string such as a very long word, by default i
...
+CSS will display overflow in this way, because doing something else could cause data loss. In CSS data loss means that some of your content vanishes. So the initial value of {{cssxref("overflow")}} is `visible`, and we can see the overflowing text. It is generally better to be able to see overflow, even if it is messy. If things were to disappear or be cropped as would happen if `overflow` was set to `hidden` you might not spot it when previewing your site. Messy overflow is at least easy to spot, and in the worst case, your visitor will be able to see and read the content even if it looks a bit strange.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:22]: to do doesn't seem to fit here

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ In CSS, if you have an unbreakable string such as a very long word, by default i
...
 
-In this next example you can see what happens if `overflow` is set to `hidden`.
+In this next example, you can see what happens if `overflow` is set to `hidden`.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:24]: a comma after the introductory phase <b>In this next example</b>

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 3]
```diff
@@ -41,21 +41,21 @@ If the box needs to be a fixed size, or you are keen to ensure that long words c
 
 > **Note:** The `overflow-wrap` property acts in the same way as the non-standard property `word-wrap`. The `word-wrap` property is now treated by browsers as an alias of the standard property.
 
-An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break even if placing the word onto a new line would allow it to display without breaking.
+An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break-even if placing the word onto a new line would allow it to display without breaking.
 
-In this next example you can compare the difference between the two properties on the same string of text.
+In this next example, you can compare the difference between the two properties on the same string of text.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:46]: a comma after the introductory phase <b>In this next example</b>

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 4]
```diff
@@ -41,21 +41,21 @@ If the box needs to be a fixed size, or you are keen to ensure that long words c
...
 
 {{EmbedGHLiveSample("css-examples/css-text/word-break-checkbox.html", '100%', 660)}}
 
 ## Adding hyphens
 
-To add hypens when words are broken, use the CSS {{cssxref("hyphens")}} property. Using a value of `auto`, the browser is free to automatically break words at appropriate hyphenation points, following whatever rules it chooses. To have some control over the process, use a value of `manual`, then insert a hard or soft break character into the string. A hard break (`‐`) will always break, even if it is not necessary to do so. A soft break (`­`) only breaks if breaking is needed.
+To add hyphens when words are broken, use the CSS {{cssxref("hyphens")}} property. Using a value of `auto`, the browser is free to automatically break words at appropriate hyphenation points, following whatever rules it chooses. To have some control over the process, use a value of `manual`, then insert a hard or soft break character into the string. A hard break (`‐`) will always break, even if it is not necessary to do so. A soft break (`­`) only breaks if breaking is needed.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:58]: missing h after p in hyphen

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 5]
```diff
@@ -41,21 +41,21 @@ If the box needs to be a fixed size, or you are keen to ensure that long words c
...
-An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break even if placing the word onto a new line would allow it to display without breaking.
+An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break-even if placing the word onto a new line would allow it to display without breaking.
 
-In this next example you can compare the difference between the two properties on the same string of text.
+In this next example, you can compare the difference between the two properties on the same string of text.
 
 {{EmbedGHLiveSample("css-examples/css-text/word-break.html", '100%', 700)}}
 
-This might be useful if you want to prevent a large gap appearing if there is just enough space for the string. Or, where there is another element that you would not want the break to happen immediately after.
+This might be useful if you want to prevent a large gap from appearing if there is just enough space for the string. Or, where there is another element that you would not want the break to happen immediately after.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:50]: Is the preposition useful here?

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 6]
```diff
@@ -41,21 +41,21 @@ If the box needs to be a fixed size, or you are keen to ensure that long words c
 
 > **Note:** The `overflow-wrap` property acts in the same way as the non-standard property `word-wrap`. The `word-wrap` property is now treated by browsers as an alias of the standard property.
 
-An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break even if placing the word onto a new line would allow it to display without breaking.
+An alternative property to try is {{cssxref("word-break")}}. This property will break the word at the point it overflows. It will cause a break-even if placing the word onto a new line would allow it to display without breaking.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:44]: maybe <b>break even> missing a hyphen here

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/css/css_text/wrapping_text/index.md [hunk 7]
```diff
@@ -31,7 +31,7 @@ To find the minimum size of the box that will contain its contents with no overf
 
 {{EmbedGHLiveSample("css-examples/css-text/min-content.html", '100%', 420)}}
 
-Using `min-content` is therefore one possibility for overflowing boxes. If it is possible to allow the box to grow to be the minimum size required for the content, but no bigger, using this keyword will give you that sizing.
+Using `min-content` is therefore one possibility for overflowing boxes. If it is possible to allow the box to grow to be the minimum size required for the content, but no bigger, using this keyword will give you that size.
```
Comment (hv48067) [inline files/en-us/web/css/css_text/wrapping_text/index.md:34]: sizing is probably the wrong verb here.

**Pull request merged**